src/parser/scanner.lex

```diff
@@ -184,6 +188,10 @@ IP_OCTET                    ([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])
 {TTL_COL}                   { return TokenType::KW_TTL_COL; }
 {TRUE}                      { yylval->boolval = true; return TokenType::BOOL; }
 {FALSE}                     { yylval->boolval = false; return TokenType::BOOL; }
+{ORDER}                     { return TokenType::KW_ORDER; }
+{ASCEND}                    { return TokenType::KW_ASCEND; }
+{ASC}                       { return TokenType::KW_ASC; }
+{DESCEND}                   { return TokenType::KW_DESCEND; }
```

Comment:
No, no , should not exist  "order by xxx  ascend" or "  "order by xxx  descend" in nGQL.  

Only exist as follows "order by xxxx  asc" or "  "order by xxx desc",    the  default is asc in oder by clause.

Reply:
ok , i'll delete them